salt/config/__init__.py

```diff
@@ -395,9 +395,9 @@
     'return_retry_timer': int,
     'return_retry_timer_max': int,
 
-    # Specify a returner in which all events will be sent to. Requires that the returner in question
-    # have an event_return(event) function!
-    'event_return': str,
+    # Specify one or more returners in which all events will be sent to. Requires that the returners
+    # in question have an event_return(event) function!
+    'event_return': list,
```

Comment:
FWIW, @terminalmage, my testing shows that it doesn't log a warning when it expects 'list' and receives a string. It does log a warning when it expects str and receives a list, which led me to make this update. 

However, I made the suggested change and retested and it continues to function correctly. Thanks for the advice.